contracts/Booster.sol

```diff
@@ -69,6 +69,18 @@ contract Booster is ReentrancyGuardUpgradeable {
         bool shutdown;
     }
 
+    struct FeeDistro {
+        address distro;
+        address rewards;
+        bytes32 executionHash;
+        bool active;
+    }
+
+    address[] public allFeeTokens;
+
+    //reward identifier -> distro, execution and virtual pool data
+    mapping(address => FeeDistro) public feeTokens;
```

Comment:
we don't need a mapping for all fee distro info for each veasset project , because we have one booster for each veasset project , so it should be a single state for fee distro , execution hash and array of fee tokens (in case there is multiple fee tokens) you could use openzipplein EnumerableSet for fee tokens , and  add new mapping for fee token => lockfee (reward address)